app/views/machines/show.html.erb

```diff
@@ -1 +1,14 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
+<section>
+  <h3>Snacks</h3>
+  <ul>
+    <% @machine.snacks.each do |snack| %>
+      <li>Snack: <%= snack.name %>, Price: $<%= snack.price %></li>
```

Comment:
To address what you wrote in your comment, you could use an action view helper called `number_to_currency` to help these prices stay at two decimal places even if they have trailing zeros.